https://bugs.gentoo.org/946510
https://github.com/Freed-Wu/pyrime/pull/3
Using Cython requires both 'cython' and 'c' to be specified in the
project().
--- a/meson.build
+++ b/meson.build
@@ -1,3 +1,3 @@
-project('pyrime', 'cython', version: '0.0.7')
+project('pyrime', 'cython', 'c', version: '0.0.7')
 
 subdir('src/pyrime')
